addons/material-forum-editor-buttons/addon.json

```diff
@@ -0,0 +1,22 @@
+{
+  "name": "Material forum editor buttons",
+  "description": "Changes the forum editor icons from tiny 16x16 bitmaps to SVG Google Material Icons.",
+  "userstyles": [
+    {
+      "url": "icons.css",
+      "matches": ["https://scratch.mit.edu/discuss/*"]
+    }
+  ],
+  "tags": ["community", "forums", "theme", "beta", "recommended"],
+  "credits": [
+  {
+    "name": "easrng",
```

Comment:
Most addons use Scratch usernames ([A-E-](https://scratch.mit.edu/users/A-E-/) in this case) in credits.